modules/kubernetes/grafana-agent/charts/grafana-agent-extras/templates/grafanaagent.yaml

```diff
@@ -28,3 +28,19 @@ spec:
       memory: {{ .Values.agent.resources.requests.memory }}
   tolerations:
     - operator: "Exists"
+  podMetadata:
+    labels:
+      aadpodidbinding: grafana-agent
+  containers:
+    - name: grafana-agent
+      volumeMounts:
+        - name: secret-store
+          mountPath: "/mnt/secret-store"
+          readOnly: true
+  volumes:
+    - name: secret-store
+      csi:
+        driver: secrets-store.csi.k8s.io
+        readOnly: true
+        volumeAttributes:
+          secretProviderClass: xenit-proxy-certificate
```

Comment:
Isn't secretProviderClass only available in a namespace?
I'm unable to find any definition of this secretProviderClass or am I missing something?

Reply:
Oh, I have it locally. Forgot to add it. On it's way